packages/graphpack/lib/server.js

```diff
@@ -28,6 +29,9 @@ const server = new ApolloServer({
   resolvers,
 });
 
+// Apply user provided middlewares
+server.applyMiddleware(applyMiddleware);
```

Comment:
Maybe let's only call this function only if there are valid middlewares?

Reply:
I'm ok with doing it if there is any middlewares available. could you please elaborate more on validating?
According to apollo-server-[integration] libs, the only required parameter for a middleware is app which should be an instance of that server type. is checking for `applyMiddleware.app` enough?

Reply:
Yep that should be okay! I’m also thinking of moving the entire config.js into backpack.config.js but that would be another PR/issue.

Reply:
That we can do on another PR :)
QQ: How would you pass multiple middlewares using this pattern? is it already handled in Apollo server or we should accept a list of middlewares?

Reply:
For that I’d stick with how Apollo is handling it, so passing the app that has all the middlewares applied.

I’ll aim to merge this once github is fixed 👍